Dockerfile

```
@@ -0,0 +1,11 @@
+# 사용할 base 이미지 선택
+FROM arm64v8/eclipse-temurin:17:jdk-focal
```

Hadolint (2.12.0)

[error] unexpected ':'
expecting '@', '\', a new line followed by the next instruction, or the image tag

(DL1000)